ocaml/idl/datamodel.ml

```diff
@@ -5173,6 +5173,9 @@ module VM_guest_metrics = struct
         ; field ~qualifier:DynamicRO
             ~ty:(Map (String, String))
             "os_version" "version of the OS"
+        ; field ~qualifier:DynamicRO
+            ~ty:(Map (String, String))
+            ~lifecycle:[] "netbios_name" "The NETBIOS name of the machine"
```

Comment:
This would need a `~default_value` as well. I'm surprised you didn't get an error about that 🤔

Reply:
added a `default_value`